web/modules/custom/ai_screening_project/src/Template/ProjectStatusEvaluationTwigExtension.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\ai_screening_project\Template;
+
+use Drupal\ai_screening_project\Helper\ProjectHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig extension providing project related functionalities.
+ *
+ * @package Drupal\ai_screening_project\Template
+ */
+class ProjectStatusEvaluationTwigExtension extends AbstractExtension {
+
+  public function __construct(
+    private readonly ProjectHelper $projectHelper,
+  ) {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getName() {
+    return 'ai_screening_project';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFunctions() {
+    return [
+      new TwigFunction('project_status_evaluation', [$this, 'getProjectTrackEvaluation']),
```

Comment:
Consider using https://www.php.net/manual/en/functions.first_class_callable_syntax.php here:

```suggestion
      new TwigFunction('project_status_evaluation', $this->getProjectTrackEvaluation(...)),
```